plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -488,12 +626,92 @@ fu_cros_ec_usb_device_send_done (FuDevice *device)
 	}
 }
 
+static gboolean
+fu_cros_ec_usb_device_send_subcommand (FuDevice *device, guint16 subcommand,
+				       gpointer cmd_body, gsize body_size,
+				       gpointer resp, gsize *resp_size,
+				       gboolean allow_less, GError **error)
+{
+	fu_cros_ec_usb_device_send_done (device);
+
+	if (!fu_cros_ec_usb_ext_cmd (device, subcommand,
+				     cmd_body, body_size,
+				     resp, resp_size, FALSE, error)) {
+		g_prefix_error (error,
+				"failed to send subcommand %" G_GUINT16_FORMAT ": ",
+				subcommand);
+		return FALSE;
+	}
+
+	/* success */
+	return TRUE;
+}
+
+static gboolean
+fu_cros_ec_usb_device_reset_to_ro (FuDevice *device, GError **error)
+{
+	gsize response_size = 1;
+	guint8 response;
+	guint16 subcommand;
+	guint8 command_body[2]; /* Max command body size. */
+	gsize command_body_size = 0;
+
+	/* send subcommand to remain in RO */
+	subcommand = UPDATE_EXTRA_CMD_STAY_IN_RO;
+	if (!fu_cros_ec_usb_device_send_subcommand (device, subcommand, command_body,
+				     command_body_size, &response,
+				     &response_size, FALSE, error))
+		return FALSE;
+
+	response_size = 1;
+	subcommand = UPDATE_EXTRA_CMD_IMMEDIATE_RESET;
+	if (!fu_cros_ec_usb_device_send_subcommand  (device, subcommand, command_body,
+				     command_body_size, &response,
+				     &response_size, FALSE, error)) {
+		/* failure here is ok */
+		g_clear_error (error);
+		return TRUE;
+	}
+
+	/* success */
+	return TRUE;
+}
+
+static gboolean
+fu_cros_ec_usb_device_jump_to_rw (FuDevice *device, GError **error)
+{
+	gsize response_size;
+	guint8 response;
+	guint16 subcommand;
+	guint8 command_body[2]; /* Max command body size. */
+	gsize command_body_size;
+
+
+	command_body_size = 0;
+	response_size =1;
+
+	subcommand = UPDATE_EXTRA_CMD_JUMP_TO_RW;
+	if (!fu_cros_ec_usb_device_send_subcommand  (device, subcommand, command_body,
+				     command_body_size, &response,
+				     &response_size, FALSE, error)) {
```

Comment:
since you're always clearing any error and discarding messages associated to it, I suggest to drop the error argument from `fu_cros_ec_usb_device_jump_to_rw` and just send `NULL` as the error argument in `fu_cros_ec_usb_device_send_subcommand`